65black/StackOlite
# #160201702 Edit Answer

#### What does this PR do?
This PR consumes the PUT `/api/v1/questions/:questionId/answers/:answerId` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Log in to an existing account.
 - Navigate to questions page.
 - Open any question and add an answer.
 - Now click on edit answer and input your new answer.
 - The page will refresh and you should see your edited answer.
 
Or if you own the question, you can also set an answer as your preferred.
 - Clone this repo and open index.html with Chrome.
 - Log in to an existing account.
 - Add your own question.
 - Add an answer to your question with another account and log out.
 - Log in with your first account and open the question.
 - You should see a button to accept answer on the answer you posted with the second account.
 - Click it and the page will refresh.
 - The answer should now have a green border.

#### What are the relevant pivotal tracker stories?
#160201702
